src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,11 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+
+        Lottery lottery = new Lottery();
```

Comment:
Don't create several instances of the same class if you can use only one instance for your purposes. You can move the 'lottery' variable declaration outside of the 'main' method and make it a class-level field.